### PR TITLE
[Oracle] Resolve divisor error in performance datastream

### PR DESCRIPTION
**Pull request by agithomas**
- Bug

## What does this PR do?

The query `SELECT name, physical_reads, db_block_gets, consistent_gets, 1 - (physical_reads / (db_block_gets + consistent_gets)) "Hit_Ratio" FROM V$BUFFER_POOL_STATISTICS` may result in error when `(db_block_gets + consistent_gets)` is zero.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

  - elastic-package build
  - elastic-package stack up -v -d --services package-registry
 

## Related issues

- Closes https://github.com/elastic/sdh-beats/issues/3762

## Screenshots

After the change

image

## PR Review Priority - High 
  The PR is related to an SDH

### CODE REVIEW

**Comment by agithomas** [comment]
Dashboard verification completed

image

File: packages/oracle/data_stream/performance/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -7,7 +7,7 @@ hosts:
 raw_data.enabled: true
 driver: "oracle"
 sql_queries: 
-  - query: SELECT name, physical_reads, db_block_gets, consistent_gets, 1 - (physical_reads / (db_block_gets + consistent_gets)) "Hit_Ratio" FROM V$BUFFER_POOL_STATISTICS
+  - query: SELECT name, physical_reads, db_block_gets, consistent_gets, 1 - (physical_reads / (db_block_gets + consistent_gets)) "Hit_Ratio" FROM V$BUFFER_POOL_STATISTICS where (db_block_gets + consistent_gets) > 0
     response_format: table
```
Comment (ishleenk17) [inline packages/oracle/data_stream/performance/agent/stream/stream.yml.hbs:11]: Having this check as a prevention mechanism is fine. 
But, in a properly configured Oracle Instance, can there be a use case where this (db_block_gets + consistent_gets) is 0?

Comment (agithomas) [reply packages/oracle/data_stream/performance/agent/stream/stream.yml.hbs:11]: The same question was asked in the SDH and it is observed in a production instance of Oracle, surprisingly.

Comment (agithomas) [reply packages/oracle/data_stream/performance/agent/stream/stream.yml.hbs:11]: Values as below is observed 

`select name, physical_reads from V$BUFFER_POOL_STATISTICS;
DEFAULT 413583949349
KEEP 0`

Comment (ishleenk17) [reply packages/oracle/data_stream/performance/agent/stream/stream.yml.hbs:11]: Ok, then this check should be fine

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Pull request merged**